buildenv/jenkins/tapVerification/tapVerification.groovy

```diff
@@ -0,0 +1,75 @@
+#!groovy
+
+def AQA_REPO = params.AQA_REPO ?: "adoptium"
+def AQA_BRANCH = params.AQA_BRANCH ?: "master"
+def JDK_VERSIONS = params.JDK_VERSIONS.trim().split("\\s*,\\s*");
+def PLATFORMS = params.PLATFORMS.trim().split("\\s*,\\s*");
+def TARGETS = params.TARGETS.trim().split("\\s*,\\s*");
+def LABEL = (params.LABEL) ?: ""
+def LABEL_ADDITION = (params.LABEL_ADDITION) ?: ""
+
+// Use BUILD_USER_ID if set and jdk-JDK_VERSIONS
+def DEFAULT_SUFFIX = (env.BUILD_USER_ID) ? "${env.BUILD_USER_ID} - jdk-${params.JDK_VERSIONS}" : "jdk-${params.JDK_VERSIONS}"
+def PIPELINE_DISPLAY_NAME = (params.PIPELINE_DISPLAY_NAME) ? "#${currentBuild.number} - ${params.PIPELINE_DISPLAY_NAME}" : "#${currentBuild.number} - ${DEFAULT_SUFFIX}"
+
+timestamps{
+    // Set the AQA_TEST_PIPELINE Jenkins job displayName
+    currentBuild.setDisplayName(PIPELINE_DISPLAY_NAME)
+    node(LABEL) {
+        TIME_LIMIT =  params.TIME_LIMIT ? params.TIME_LIMIT.toInteger() : 1
+        timeout(time: TIME_LIMIT, unit: 'HOURS') {
+            // change openjdk-tests to aqa-tests
+            sh "curl -Os https://raw.githubusercontent.com/${AQA_REPO}/openjdk-tests/${AQA_BRANCH}/buildenv/jenkins/tapVerification/aqaTap.sh"
+            sh "chmod 755 aqaTap.sh"
```

Comment:
need to verify the file after download.

Reply:
If the file does not exist, chmod will have error `chmod: aqaTap.sh: No such file or directory`. Also, this is a file in git repo, so the file can be updated. If we check the file sha, then we have to update the sha for each file update.